changelog.d/20230830_153211_30907815+rjmello_self_diagnostic_env.rst

```diff
@@ -0,0 +1,7 @@
+New Functionality
+^^^^^^^^^^^^^^^^^
+
+- The `self-diagnostic` endpoint command is now aware of the `GLOBUS_SDK_ENVIRONMENT`
+  environment variable and will test connectivity to the relevant services.
+
+- Added web service version information to the output of the `self-diagnostic` endpoint command.
```

Comment:
A friendly reminder that this is reST, not Markdown.  (double backticks ...)

Reply:
Ah, nice catch.